exporter/elasticsearchexporter/README.md

```diff
@@ -78,7 +78,20 @@ All other defaults are as defined by [confighttp].
 
 ### Queuing
 
-The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. However, the sending queue is currently disabled by default.
+The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. The sending queue is enabled by default.
+
+Default `num_consumers` is `100`. 
+
+When persistent queue is used, there should be no event loss even on collector crashes.
+
+### Batching
+
+The Elasticsearch exporter supports the common `batcher` settings.
```

Comment:
We should definitely mention here that the batcher config is experimental and may change without notice.